pymedphys/_dicom/delivery/core.py

```diff
@@ -161,20 +163,28 @@ def _from_dicom_beam(cls, beam, meterset):
         )
 
         dicom_mlcs = rtplan.get_leaf_jaw_positions_for_type(
-            beam_limiting_device_position_sequences, "MLCX"
+            beam_limiting_device_position_sequences, "MLC"
         )
 
-        mlcs = [
-            np.array(
-                [-np.array(mlc[0:num_leaves][::-1]), np.array(mlc[num_leaves::][::-1])][
-                    ::-1
-                ]
-            ).T
-            for mlc in dicom_mlcs
-        ]
+        mlcs = np.array(
+            [
+                np.array(
+                    [
+                        -np.array(mlc[0:num_leaves][::-1]),
+                        np.array(mlc[num_leaves::][::-1]),
+                    ][::-1]
+                ).T
+                for mlc in dicom_mlcs
+            ]
+        )
+
+        # Nasty hack, patching the fact that the leaf banks are swapped
+        # around for MLCY vs MLCX
+        if "MLCY" in rt_beam_limiting_device_types:
+            mlcs[:, :, [0, 1]] = mlcs[:, :, [1, 0]]
 
         dicom_jaw = rtplan.get_leaf_jaw_positions_for_type(
-            beam_limiting_device_position_sequences, "ASYMY"
+            beam_limiting_device_position_sequences, "ASYM"
```

Comment:
I have concerns about using a value that isn't a defined or enumerated CS in the DICOM Standard .

Reply:
Yup, this is hand in hand with the comment above:

https://github.com/pymedphys/pymedphys/pull/1120#discussion_r517044617

Will fix this.

Reply:
Issue specified in https://github.com/pymedphys/pymedphys/issues/1139